ani-cli

```diff
@@ -350,7 +350,7 @@ quality="${ANI_CLI_QUALITY:-best}"
 case "$(uname -a | cut -d " " -f 1,3-)" in
     *Darwin*) player_function="${ANI_CLI_PLAYER:-$(where_iina)}" ;;   # mac OS
     *ndroid*) player_function="${ANI_CLI_PLAYER:-android_mpv}" ;;     # Android OS (termux)
-    *MINGW* | *WSL2*) player_function="${ANI_CLI_PLAYER:-mpv.exe}" ;; # Windows OS
+    *MINGW* | *WSL2*) player_function="${ANI_CLI_PLAYER:-mpv}" ;;     # Windows OS
```

Comment:
This will make WSL use the mpv installed in WSL (which is what you wanted probably), but it'll break git bash because it also matches with `*MINGW*`, and it won't find mpv (but does find mpv.exe)

A better solution would be to remove the `*WSL2*` option and let it default to `mpv`, or even better for you to export the env var `ANI_CLI_PLAYER` as `mpv`